wandb/env.py

```diff
@@ -378,7 +378,7 @@ def get_data_dir(env: Optional[Env] = None) -> str:
 
 
 def get_artifact_dir(env: Optional[Env] = None) -> str:
-    default_dir = os.path.join(".", "artifacts")
+    default_dir = os.path.abspath(os.path.join(".", "artifacts"))
```

Comment:
Nit: do this on line 385 so that we also do it for user provided paths.

Reply:
oh yes, good catch. tysm